back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -475,6 +478,35 @@ def resolve_add_packing_list_entry_to_distribution_event(*_, creation_input):
     )
 
 
+@mutation.field("updatePackingListEntry")
+@convert_kwargs_to_snake_case
+def resolve_update_packing_list_entry(*_, packing_list_entry_id, number_of_items):
+    authorize(permission="packing_list_entry:write")
+    return update_packing_list_entry(g.user.id, packing_list_entry_id, number_of_items)
+
+
+@mutation.field("removeAllPackingListEntriesFromDistributionEventForProduct")
+@convert_kwargs_to_snake_case
+def resolve_remove_all_packing_list_entries_from_distribution_event_for_product(
+    *_, distribution_event_id, product_id
+):
+    authorize(permission="packing_list_entry:write")
```

Comment:
Does this action require `distro_event:write` permission as well?

Reply:
No, I think packing_list_entry:write is enough